docs/reference/search/field-stats.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[search-field-stats]]
+== Field stats API
+
+experimental[]
+
+The field stats api allows to easily figure certain statistical properties without a field without executing a search, but
+looking up measurements that are natively available in the Lucene index. This can be useful to explore a data set,
+that you don't know much about in order to for example create histogram aggregation with a meaningful interval.
+
+The field stats api by defaults executes on all indices, but can execute on specific indices too.
+
+All indices:
+
+[source,js]
+--------------------------------------------------
+curl -XGET "http://localhost:9200/_field_stats?fields=rating"
+--------------------------------------------------
+
+Specific indices:
+
+[source,js]
+--------------------------------------------------
+curl -XGET "http://localhost:9200/index1,index2/_field_stats?fields=rating"
+--------------------------------------------------
+
+Supported request option:
+* `fields` - A list of fields to compute stats for.
+
+==== Field statistics
+
+The field stats api is supported on string based, number based and date based fields and can return the following statistics per field:
+
+* `max_doc`             - The total number of documents.
+* `doc_count`           - The number of documents that have at least one term for this field, or -1 if this measurement isn't available on one or more shards.
+* `density`             - A ratio as a percentage between all the documents that have at least one value for this field and all documents.
```

Comment:
maybe we can reword the description when pushing, like "The percentage of documents that have at least one value for this field"